tests/test_matrix_utils.py

```
@@ -3559,6 +3563,86 @@ def fake_async_client(*_args, **_kwargs):
     mock_client.close.assert_awaited_once()
 
 
+@pytest.mark.asyncio
+async def test_connect_matrix_sync_validation_error_retries_with_invite_safe_filter(
+    monkeypatch,
```

Ruff (0.14.14)

Unused function argument: `monkeypatch`

(ARG001)